demos/common/cpp/utils/include/utils/ocv_common.hpp

```diff
@@ -188,7 +188,6 @@ static inline void resize2tensor(const cv::Mat& mat, const ov::Tensor& tensor) {
     cv::Size size{int(shape[ov::layout::width_idx(layout)]), int(shape[ov::layout::height_idx(layout)])};
     assert(tensor.get_element_type() == ov::element::u8);
     assert(shape.size() == 4);
-    assert(shape[ov::layout::batch_idx(layout)] == 1);
```

Comment:
What is the actual value here?

Reply:
it does not work in case dyn batch. does not work for interactive_face_detection too, you should add test case

Reply:
well, even though test case for dyn batch might take time, but better to check this condition locally at least. batch is zero for the first iteration of dyn batch and raise assertion in debug build

Reply:
> batch is zero

This is an ov bug then. `Tensor::get_size()` is defined as N*H*W*C. If any of dimensions is zero, than overall size is also zero. Please report the bug

Reply:
I've no time for that. You may fill bug based on interactive_face_detection_demo

Reply:
You should be able to revert this after applying https://github.com/openvinotoolkit/open_model_zoo/pull/2923#discussion_r802666212

Reply:
reverted